pom.xml

```diff
@@ -216,6 +217,64 @@
     </build>
 
     <profiles>
+		<profile>
+			<id>rpm</id>
+			<build>
+				<plugins>
+					<plugin>
+						<groupId>org.codehaus.mojo</groupId>
+						<artifactId>rpm-maven-plugin</artifactId>
+						<version>2.1.5</version>
+						<executions>
+							<execution>
+								<id>generate-rpm</id>
+								<goals>
+									<goal>rpm</goal>
+								</goals>
+							</execution>
+						</executions>
+						<configuration>
+							<group>Applications/Internet</group>
+							<packager>Confluent Packaging</packager>
+							<license>${licenses.name}</license>
+							<needarch>noarch</needarch>
+							<targetOS>linux</targetOS>
+							<requires>
+								<require>confluent-common &gt;= ${project.version}</require>
```

Comment:
Based on discussion around the top level confluent-platform RPMs allowing use of a newer bug fix release of its dependencies, do we want to switch these to be just `=`?